features/identity-event/org.wso2.carbon.identity.event.server.feature/resources/identity-event.properties

```diff
@@ -27,6 +27,7 @@ account.lock.handler.enable=false
 account.lock.handler.login.fail.timeout.ratio=2
 account.lock.handler.On.Failure.Max.Attempts=5
 account.lock.handler.Time=5
+account.lock.handler.notification.internally.manage=true
```

Comment:
I think the name of the property should be changed as below for the sake of consistensy

account.lock.handler.notification.internally.manage ==> account.lock.handler.notification.manageInternally.